clustering/ejb/WFLY-14953_Create_distributable-ejb_subsystem.adoc

```diff
@@ -0,0 +1,298 @@
+        <cache name="distributable-cache" passivation-store="infinispan"/>
+    </caches>
+    ...
+    <passivation-stores>
+      <passivation-store name="infinispan" cache-container="ejb" max-sze="10000">
+    </passivation-stores>
+    ...
+</subsystem>
+----
+With this RFE, the same cache factories are configured as follows:
+----
+<subsystem xmlns="urn:jboss:domain:ejb3:10.0">
+     ...
+     <caches>
+        <!-- a non-passivating cache factory -->
+        <simple-cache name="simple-cache"/>
+        <!-- a passivating cache factory with its bean management provider -->
+        <distributable-cache name="distributable-cache" bean-management="infinispan"/>
+    </caches>
+    ...
+  </subsystem>
+----
+The bean management provider is now configured separately in the distributable-ejb subsystem:
+----
+<subsystem xmlns="urn:jboss:domain:distributable-ejb:1.0" default-bean-management="infinispan">
+    <infinispan-bean-management name="infinispan" cache-container="ejb" cache="dist" max-size="10000"/>
+    ...
+</subsystem>
+----
+
+=== client mappings registries
+
+Prior to this RFE, client mappings registries were configured as follows:
+----
+<subsystem xmlns="urn:jboss:domain:ejb3:10.0">
+    ...
+    <remote cluster="ejb" connectors="http-remoting-connector" thread-pool-name="default">
+       <channel-creation-options>
+            <option name="MAX_OUTBOUND_MESSAGES" value="1234" type="remoting"/>
+       </channel-creation-options>
+    </remote>
+    ...
+</subsystem>
+----
+Here, the "cluster" attribute identifies the group name of the infinispan cache backing the client mappings
+registry, used to store  client mappings for this server. The implementation of the client mappings registry is
+hard-wired into the _ejb3_ subsystem.
+
+With this RFE, client mappings registries are configured as follows:
+----
+<subsystem xmlns="urn:jboss:domain:ejb3:10.0">
+    ...
+    <remote connectors="http-remoting-connector" thread-pool-name="default">
+       <channel-creation-options>
+            <option name="MAX_OUTBOUND_MESSAGES" value="1234" type="remoting"/>
+       </channel-creation-options>
+    </remote>
+    ...
+</subsystem>
+----
+The "cluster" attribute is now removed from the remote element of the _ejb3_ subsystem, and is now instead
+specified in the client mappings registry element of the _distributable-ejb_ subsystem:
+----
+<subsystem xmlns="urn:jboss:domain:distributable-ejb:1.0" default-bean-management="infinispan">
+    ...
+    <infinispan-client-mappings-registry cache-container="ejb" cache="client-mappings"/>
+</subsystem>
+----
+
+To ensure that legacy configurations (configurations written for _ejb3_ subsystem schema 9 or lower) still work with
+a server incorporating this RFE, the legacy elements described above will be accepted as part of a valid configuration,
+and legacy providers will be used to guarantee the same legacy functionality. The only restriction here is that
```

Comment:
Regarding `The only restriction here is that...` let's be sure that this detail will appear in the community docs.

Reply:
The community documentation will not describe legacy configuration, as its usage is to be discouraged (as these resources will be removed in a later release).

Reply:
Hi @pferraro - and thanks for your feedback. It seems valid as a general principle but I am thinking that a user would have to look into this proposal in order to know about this restriction. Anyways, given what you stated, that seems reasonable as well, so I'll resolve this thread.

Reply:
> I am thinking that a user would have to look into this proposal in order to know about this restriction

Not at all. The management model has formal deprecation support, and, as a general rule, deprecated resources/attributes/operations to indicate their replacements in their deprecation messages, which appear in the logs whenever a deprecated resource/attribute/operation is used.